proposal.gemspec

```diff
@@ -16,9 +16,7 @@ Gem::Specification.new do |s|
   s.files = Dir["{app,config,db,lib}/**/*"] + ["Rakefile"]
   s.test_files = Dir["test/**/*"]
 
-  s.add_dependency "rails", "~> 4.0"
+  s.add_dependency "rails", ">= 5.0.0.beta3", "< 5.1"
```

Comment:
I've already update this line of code. We can change this to ```~> 5.0``` once it's released, probably in v2.0.1